source/kraken/maintenance_worker.cpp

```diff
@@ -508,7 +547,13 @@ void MaintenanceWorker::listen_rabbitmq() {
         size_t max_batch_nb = conf.broker_max_batch_nb();
 
         try {
+            auto begin_rt_retrieval = pt::microsec_clock::universal_time();
             auto rt_envelopes = consume_in_batch(rt_tag, max_batch_nb, timeout_ms, no_ack);
+            auto duration_rt_retieval = pt::microsec_clock::universal_time() - begin_rt_retrieval;
```

Comment:
```suggestion
            auto duration_rt_retrieval = pt::microsec_clock::universal_time() - begin_rt_retrieval;
```

Reply:
Done in https://github.com/hove-io/navitia/pull/3955/commits/e7122af1bdb0aaec2c912e7f5dde1bb3f60b9c68